modules/stochastic_tools/doc/content/source/surrogates/PolynomialChaos.md

```diff
@@ -42,20 +42,40 @@ The weighting functions are defined by the probability density function of the p
 | Exponential | $e^{-\xi}$ | Laguerre | $[0,\infty]$ |
 | Gamma | $\frac{\xi^{\alpha}e^{-\xi}}{\Gamma(\alpha+1)}$ | Generalized Laguerre | $[0,\infty]$ |
 
-The expression in [eq:coeff] can be integrated using many different techniques. One is performing a Monte Carlo integration,
+## Computing Coefficients
 
-!equation
-q_i = \frac{1}{\left\langle\Phi_i(\vec{\xi}),\Phi_i(\vec{\xi})\right\rangle} \frac{1}{N_{\mathrm{mc}}}\sum_{n=1}^{N_{\mathrm{mc}}} Q(\vec{\xi}_n)\Phi_i(\vec{\xi}_n) ,
+There are currently two techniques to compute the coefficients in [!eqref](eq:expansion), the first performs the integration described by [!eqref](eq:coeff). For Monte Carlo sampling, the integration is in the form
 
+!equation
+q_i = \frac{1}{\left\langle\Phi_i(\vec{\xi}),\Phi_i(\vec{\xi})\right\rangle} \frac{1}{N_{\mathrm{mc}}}\sum_{n=1}^{N_{\mathrm{mc}}} Q(\vec{\xi}_n)\Phi_i(\vec{\xi}_n) .
 
-or using numerical quadrature,
+And for using a [QuadratureSampler.md]
 
 !equation
 q_i = \frac{1}{\left\langle\Phi_i(\vec{\xi}),\Phi_i(\vec{\xi})\right\rangle}\sum_{n=1}^{N_q} w_n Q(\vec{\xi}_n)\Phi_i(\vec{\xi}_n) .
 
-
 The numerical quadrature method is typically much more efficient that than the Monte Carlo method and has the added benefit of exactly integrating the polynomial basis. However, the quadrature suffers from the curse of dimensionality. The naive approach uses a Cartesian product of one-dimensional quadratures, which results in $(\max(k^d_i) + 1)^D$ quadrature points to be sampled. Sparse grids can help mitigate the curse of dimensionality significantly.
 
+The other technique is using ordinary least-squares (OLS) regression, like in [PolynomialRegressionTrainer.md]. This is generally more accurate than integration for Monte Carlo sampling, as shown in the figure below.
```

Comment:
Hmmm the accuracy might depend on the dimensionality, basis functions and the function we are trying to approximate. Unless we can cite a proof here, maybe 'generally' could be replaced by 'in many cases/instances'.